en/config/actuators.md

```diff
@@ -81,7 +76,7 @@ Note, this may not be the same as the position of the flight controller!
 
 #### Motor Geometry: VTOL Quadrotor Tailsitter
 
-The motor geometry for a [VTOL Quad Tailsitter](../airframes/airframe_reference.md#vtol-quad-tailsitter) is shown below (the approach for configuring other tailsitter VTOL vehicles will be similar).
+The motor geometry for a [VTOL Quad Tailsitter](../airframes/airframe_reference.md#vtol-quad-tailsitter) is shown below (the approach for configuring other tailsitter VTOL vehicles will be similar). 
```

Comment:
@sfuhrer There does not appear to be a VTOL Quad Tailsitter config any more. IN the actuators doc we refer to that here. Can I just replace with the generic quad tailsitter config at https://docs.px4.io/main/en/airframes/airframe_reference.html#vtol-tailsitter ?

Reply:
Yes :+1: